testing/kokoro/continuous-v8.cfg

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Location of the build script in this repository.
+build_file: "cloud-profiler-nodejs/testing/v8_integration_test.sh"
```

Comment:
I would name the file v8_canary_integration_test.sh and the enclosing file continuous-v8-canary.cfg. Otherwise it's not clear (everything is v8-based in Node.js, at large).

Reply:
Done